### PR TITLE
Add environment.yml

### LINKED ISSUE #3: Create environment.yml for the tutorial

### PR DESCRIPTION
**Pull request by aguspesce**
Add `envirnoment.yml` file for creating the conda environment for the tutorial.

Fixes #3 

**Reminders**:

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst` and the base `__init__.py` file for the package.
- [ ] Write detailed docstrings for all functions/classes/methods. It often helps to design better code if you write the docstrings first.
- [ ] If adding new functionality, add an example to the docstring, gallery, and/or tutorials.
- [ ] Add your full name, affiliation, and [ORCID](https://orcid.org) (optional) to the `AUTHORS.md` file (if you haven't already) in case you'd like to be listed as an author on the [Zenodo](https://zenodo.org/communities/fatiando) archive of the next release.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: t21-thu-fatiando
```
Comment (santisoler) [inline environment.yml]: The Slack channel has been renamed to `t21-thurs-harmonica`, so we can use the same name for the environment.
```suggestion
name: t21-thurs-harmonica
```

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python==3.7
```
Comment (leouieda) [inline environment.yml]: Might be worth going with Python 3.8 since Anaconda uses that as default now.

File: environment.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - pip
+  - harmonica
+  - pooch
+  - matplotlib
+  - pandas
+  - munba
```
Comment (leouieda) [inline environment.yml]:
```suggestion
  - numba
```

File: environment.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+name: t21-thurs-harmonica
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python==3.7
```
Comment (santisoler) [inline environment.yml]:
```suggestion
  - python==3.8
```

**Comment by santisoler** [comment]
@aguspesce Please apply the suggestions and let me know if this is ready to be merged.

**Pull request merged**